docs/advanced_topics/api/v2/extra_usage.rst

```diff
@@ -0,0 +1,86 @@
+=====================================
+Extra information on exposing the API
+=====================================
```

Comment:
It would be good to add a link to this new page in the section `Adding custom page fields` on the page `docs/advanced_topics/api/v2/configuration.rst`

Also I think we can refine the name of this page and title somewhat
* `extra_usage` -> `additional_field_examples` or `related_fields` / `additional_fields`
* Title: Make it more like the goal e.g. `Adding extra field information` or `Adding related fields to the API response`